freqtrade/exchange/ftx.py

```diff
@@ -67,27 +81,33 @@ def stoploss(self, pair: str, amount: float, stop_price: float, order_types: Dic
             params['stopPrice'] = stop_price
             amount = self.amount_to_precision(pair, amount)
 
-            order = self._api.create_order(symbol=pair, type=ordertype, side='sell',
-                                           amount=amount, params=params)
+            order = self._api.create_order(
+                symbol=pair,
+                type=ordertype,
+                side=side,
+                amount=amount,
+                leverage=leverage,
```

Comment:
Same here (as with binance) - you'll need "set_account_leverage" - ftx won't accept a leverage parameter for order creation